packages/bsky/src/services/feed/views.ts

```diff
@@ -217,6 +239,65 @@ export class FeedViews {
     }
   }
 
+  userCanReply(
```

Comment:
We still have `feedutils.violatesThreadGate()` hanging around too.  I realize it might be kind of a pain to DRY these up due to the way one is tied-in with indexing logic.  Maybe for the time being we can just leave a comment so that anyone working on one will be aware of the other.